docs/concepts/index.md

```diff
@@ -10,27 +10,31 @@ tags:
 # Explore Prefect concepts
 
 ## Develop
-* [Flows](flows.md)
-* [Tasks](tasks.md)
-* [Results](results.md)
-* [Artifacts](artifacts.md)
-* [States](states.md)
-* [Blocks](blocks.md)
-* [Variables](variables.md)
-* [Task Runners](task-runners.md)
-* [Runtime Context](runtime-context.md)
-* [Profiles & Configuration](settings.md)
 
-##  Deploy
-* [Deployments](deployments.md)
-* [Projects](projects.md)
-* [Work Pools, Workers & Agents](work-pools.md)
-* [Storage](storage.md)
-* [Filesystems](filesystems.md)
-* [Infrastructure](infrastructure.md)
-* [Schedules](schedules.md)
-* [Logging](logs.md)
+| Keyword                                     | Description                                                                                                                                                                        |
+| ------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| __[Flows](flows.md)__                       | The most basic Prefect object                                                                                                                                                      |
+| __[Tasks](tasks.md)__                       | Discrete units of work in a Prefect workflow                                                                                                                                       |
+| __[Results](results.md)__                   | represent the data returned by a flow or a task                                                                                                                                    |
+| __[Artifacts](artifacts.md)__               | are persisted outputs such as tables, files, or links                                                                                                                              |
```

Comment:
```suggestion
| __[Artifacts](artifacts.md)__               | Persisted outputs for human viewing such as tables or links.                                                                                                                              |
```

Reply:
@discdiver I don't know why but "persisted outputs for human viewing" made me laugh 😄 

Thoughts on 
"Formatted outputs rendered in the Prefect UI, such as markdown, tables, or links."
?